esp/services/esdl_svc_engine/esdl_monitor.cpp

```diff
@@ -498,12 +492,12 @@ class CEsdlMonitor : implements IEsdlMonitor, public CInterface, implements IEsd
                 return LINK(procpt);
             }
             //Otherwise check if there's binding configured on the same port
-            xpath.clear().appendf("EspBinding[@type='EsdlBinding'][@port='%s'][1]", port);
+            xpath.clear().appendf("EspBinding[@type='EsdlBinding'][@port=%s][1]", port);
```

Comment:
fixed.

Reply:
As for the port, yes since port is numeric, comparing numerically will help with cases when someone manually edits the environment.xml file and uses "00" instead "0" for example.